substrate/substrate-processor/src/mapping.ts

```diff
@@ -219,6 +220,15 @@ export class Call {
         this.#subcalls = calls
     }
 
+    get siblings(): Call[] {
```

Comment:
this new field looks a bit unnatural. You already can achieve the same by using `call.parent.subcalls`